test/core/number/cos.wren

```diff
@@ -0,0 +1,6 @@
+IO.print(0.cos)             // expect: 1
+IO.print(Num.pi.cos)        // expect: -1
+IO.print((2*Num.pi).cos)    // expect: 1
```

Comment:
> Are you really sure we can't rely on precise results?

The floating-point representation itself is non-precise by definition (`pi` itself won't be represented exactly, not being an internally detected constant that changes/fixes the trigonometric functions result).

Moreover, some algorithms are more (or less) stable than others and the result diverges more (or less). Using an *epsilon* threshold is a common practice in numerical analysis.